common/lib/xmodule/xmodule/templates/html/image_modal.yaml

```diff
@@ -1,7 +1,23 @@
 ---
 metadata:
-    display_name: Full Screen Image
+    display_name: Full Screen Image Tool
 data: |
-        <h2>Full Screen Image</h2>
-        <p>Use the full screen image tool to allow students to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
-        <a href="http://static.class.stanford.edu/stanford-hills-big.jpg" class="modal-content"><img alt="The Stanford Hills" src="http://static.class.stanford.edu/stanford-hills-small.jpg" /></a>
\ No newline at end of file
+        <h2>Full Screen Image Tool</h2>
+        <p>Use the Full Screen Image tool to allow learners to open the image in the full screen, zoom in on the image, and drag the image within the screen.</p>
+        <p>With the Full Screen Image tool, learners can see the image's details as well as its context within the unit.</p>
+        <p>You enable learners to view the image in the full screen by wrapping the image in a link to the larger version of the image. The HTML format required to use the Full Screen Image tool is:</p>
+        <pre>
+        &lt;a href="<i>path to full screen image</i>" class="modal-content"&gt;
```

Comment:
are you using italics to indicate a variable here? if so, maybe do that consistently in the iFrame example too?